src/main/java/com/rocketden/main/service/DefaultCodeGeneratorService.java

```diff
@@ -0,0 +1,23 @@
+package com.rocketden.main.service;
+
+import java.util.List;
+
+import com.rocketden.main.model.Language;
+import com.rocketden.main.model.problem.ProblemIOType;
+import com.rocketden.main.model.problem.ProblemInput;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public interface DefaultCodeGeneratorService {
```

Comment:
nit: Move this into the generators folder

Reply:
Done ✅